pkg/karmadactl/cmdinit/karmada/check.go

```diff
@@ -0,0 +1,37 @@
+package karmada
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+	apiregistrationv1 "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+	aggregator "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset"
+)
+
+func waitAPIServiceReady(c *aggregator.Clientset, name string, timeout int) error {
```

Comment:
```suggestion
func waitAPIServiceReady(c *aggregator.Clientset, name string, timeout time.Duration) error {
```
Why not use the `time.Duration`?

Reply:
copy from here:  https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/cmdinit/kubernetes/check.go#L91

Is there any specification for this? If there is, I am happy to modify them together( i am golang beginer)